_posts/archived/2005-05-30-thereisnoperfectdesign.aspx.markdown

```diff
@@ -8,7 +8,6 @@ categories:
 - code
 redirect_from:
   - /archive/2005/05/29/thereisnoperfectdesign.aspx
-  - /archive/2005/05/30/thereisnoperfectdesign.aspx
```

Comment:
The problem as I understand it is a directory vs file issue:

1. jekyll-redirect-from creates `_site/archive/2005/05/30/thereisnoperfectdesign.aspx/index.html` as the redirect page
2. jekyll comes later and tries to create a **file** called `_site/archive/2005/05/30/thereisnoperfectdesign.aspx`
3. 💥  file and directory cannot share a name 💥 

/cc @benbalter who helped with the recent rewrite of jekyll-redirect-from